.changeset/spotty-dingos-push.md

```diff
@@ -0,0 +1,5 @@
+---
+"@rocket.chat/meteor": patch
+---
+
+Fixed an issue where users were not able to login using OAuth without having the 'manage-oauth-apps' permissions
```

Comment:
```suggestion
Fixed an issue where users were not able to login using OAuth Apps without having the 'manage-oauth-apps' permissions
```

Reply:
I also think we should also note the new permission here on the changeset as well